### PR TITLE
fix error prevs commit

### PR DESCRIPTION
**Pull request by pfi79**

### CODE REVIEW

**Review by yacovm — CHANGES_REQUESTED** [review]
1. Can you add a commit message? 
2. Can you please add a test that checks that when we send pre-prepare, we don't only attach the previous commit signatures when periodical leader rotation is enabled?

**Comment by pfi79** [comment]
> 1. Can you add a commit message?
> 2. Can you please add a test that checks that when we send pre-prepare, we don't only attach the previous commit signatures when periodical leader rotation is enabled?

done

File: internal/bft/view_test.go [hunk 1]
```diff
@@ -145,6 +145,98 @@ func TestViewBasic(t *testing.T) {
...
+		},
+	} {
+		t.Run(testCase.description, func(t *testing.T) {
+			basicLog, err := zap.NewDevelopment()
+			assert.NoError(t, err)
+			log := basicLog.Sugar()
+			comm := &mocks.CommMock{}
+			commWG := sync.WaitGroup{}
+			comm.On("BroadcastConsensus", mock.Anything).Run(func(args mock.Arguments) {
+				fmt.Println("Sending", args.Get(0))
```
Comment (yacovm) [inline internal/bft/view_test.go]: please remove the print

Comment (pfi79) [reply internal/bft/view_test.go]: Ugh, my inattention.

Fixed it.

**Review by yacovm — APPROVED** [review]

**Pull request merged**